src/Standards/Generic/Sniffs/Files/LowercasedFilenameSniff.php

```diff
@@ -44,7 +44,7 @@ public function process(File $phpcsFile, $stackPtr)
     {
         $filename = $phpcsFile->getFilename();
         if ($filename === 'STDIN') {
-            return $phpcsFile->numTokens;
+            return $phpcsFile->numTokens; // @codeCoverageIgnore
```

Comment:
Please remove the ignore annotation. This is testable. It just isn't tested (yet).